diagnostics/seaice_suite/seaice_suite_sic_mean_sigma.py

```diff
@@ -0,0 +1,382 @@
+    allstats
+
+    allstats.to_netcdf(file_out)
+
+
+
+
+if processmod:
+    processandsave(field,proc_mod_file,firstyr,lastyr)
+
+if processobs:
+    processandsave(obs,proc_obs_file,firstyr,lastyr)
+
+
+
+
+### 4) Read processed data, regrid model to obs grid, plot, saving figures: #######################################
+
+obsstats = xr.open_dataset(proc_obs_file)
+obsstats=obsstats.rename({'latitude':'lat'})
+obsstats=obsstats.rename({'longitude':'lon'})
+
+obsmean=obsstats.themean 
+obsstd=obsstats.thestd
+obstrend=obsstats.trend 
+obsdetrendedstd=obsstats.detrended_std 
+obsonemolagcorr=obsstats.onemolagcorr
+obsoneyrlagcorr=obsstats.oneyrlagcorr
+
+modstats = xr.open_dataset(proc_mod_file)
+
+coords = [a for a in modstats.coords]
+if 'longitude' in coords:
+    modstats=modstats.rename({'latitude':'lat'})
+    modstats=modstats.rename({'longitude':'lon'})
+
+# regrid model data to obs grid
+method = 'nearest_s2d'
+regridder = xe.Regridder(modstats, obsstats, method, periodic=False, reuse_weights=False)
+modstats=regridder(modstats)
+
+modmean=modstats.themean 
+modstd=modstats.thestd
+modtrend=modstats.trend 
+moddetrendedstd=modstats.detrended_std 
+modonemolagcorr=modstats.onemolagcorr
+modoneyrlagcorr=modstats.oneyrlagcorr
+
+month=modstats.month.values
+monthabbrev=modstats.monthabbrev.values
+
+obsstats.close()
+modstats.close()
+
+
+
+
+
+def monthlyplot(field, obs=None, edgec=None, figfile='./figure.png',
+                cmapname='PuBu_r', statname='Mean', unitname='Fraction',vmin=0.,vmax=1.):
+    fig = plt.figure(figsize=(12,10))
+    cmap_c = cmapname
+
+    if obs is None:
+        edge=False
+    else:
+        edge=True
+        if edgec==None:
+            edgec='yellow'
+            
+    for m, themonth in enumerate(monthabbrev):
+        ax = plt.subplot(3,4,m+1,projection = ccrs.NorthPolarStereo())
+        ax.add_feature(cfeature.LAND,zorder=100,edgecolor='k',facecolor='darkgrey')
+
+        ax.set_extent([0.005, 360, 50, 90], crs=ccrs.PlateCarree())
+        pl = field.sel(month=m).plot(x='lon', y='lat', vmin=vmin, vmax=vmax, 
+                        transform=ccrs.PlateCarree(),cmap=cmap_c,add_colorbar=False)
+        if edge:
+            obs.sel(month=m).plot.contour(levels=[.15],x='lon', y='lat', linewidths=2, 
+                        transform=ccrs.PlateCarree(),colors=[edgec])
+  
+        ax.set_title(themonth,fontsize=14)
+
+
+    fig.suptitle(modelname+' '+statname+' Sea Ice Concentration '+str(firstyr)+'-'+str(lastyr), fontsize=18)
+
+    cbar_ax = fig.add_axes([0.315, 0.08, 0.4, 0.02]) #[left, bottom, width, height]
+    cbar = fig.colorbar(pl, cax=cbar_ax,  orientation='horizontal')
+    
+  
+    cbar.ax.set_title(unitname,fontsize=14)
+    cbar.ax.tick_params(labelsize=12)
+    plt.subplots_adjust(bottom=0.15)
+    plt.savefig(figfile, format='png',dpi=300)
+    plt.show()
+    plt.close()
+    return
+
+
+
+
+figfile=figures_dir+'themean_'+firstyr+'-'+lastyr+'.png'
```

Comment:
Consider using Python's f-string functionality instead of string concatenation  to help with readability.  See https://docs.python.org/3/reference/lexical_analysis.html#f-strings